### DISCUSSION TITLE
Suggestion for Current Status page

### DISCUSSION THREAD

**Discussion by jrnewton** [zopencommunity/meta#349] [Porting]
The [Current Status page](https://zosopentools.github.io/meta/#/Progress) shows the % of tests completed successfully.  It would be great if you can surface the number of tests as well.  In my mind there is a big difference between 100% success rate for 2 tests vs. 2000 tests. 

Side question - am I correct in assuming that your build process is just doing a "make test"?

**Comment by IgorTodorovskiIBM** [comment]
Thanks for the suggestion, we currently report the total number of tests in this page: https://zosopentools.github.io/meta/#/Latest (although something recently just went wrong with the table formatting...). We could probably add another table with the test details in the status page.

Yep, for the most part it's make test, unless they're using a different target or build system.

**Comment by jrnewton** [comment]
Ahhh. Fixing that page would be good!  

I may still be confused though.  Looking at the vimport test results in Latest.md shows ```(1 tests pass out of 1 tests - 100.00% success rate)```.  I know for a fact that Vim has a lot of tests, although I don't remember how they are executed...  That leaves me scratching my head.

**Comment by jrnewton** [comment]
Hi @IgorTodorovskiIBM  - 

Any idea why only 1 test is run for Vim?  See my comment above.

**Reply by IgorTodorovskiIBM** [reply]
Good question, I think at the time we weren't able to get the tests going so we created a simple `vim --version` check. I'll open an issue to get the tests going. I think the `make test` target should get them going

**Reply by IgorTodorovskiIBM** [reply]
Oh we already have one here: https://github.com/ZOSOpenTools/vimport/issues/5

**Comment by MikeFultonDev** [comment]
Closing this off because we have the 'current status' and have an open issue for vim lack of testing.
